build_scripts/build-script-files/jxcore/build_scripts/android_compile.sh

```diff
@@ -0,0 +1,222 @@
+#!/bin/bash
+
+# Copyright & License details are available under JXCORE_LICENSE file
```

Comment:
Can you please put in a note here basically explaining what you explaining in the PR's description so that future folks wondering why we have two build files and where NPM is will have some hope of understanding what happened?